src/ai/.x/templates/openai-asst-webpage-ts/src/index.ts

```diff
@@ -0,0 +1,581 @@
+import { marked } from "marked";
```

Comment:
There are a still some missing types in this file. Do you want me send a PR to add those?

Reply:
The .vscode files? i just removed that part of the README.md as the other "webpage" samples don't have that set up... Ifyou want to do a separate PR to add that, that'd be great.